pyproject.toml

```diff
@@ -25,6 +25,7 @@ ignore = ["ANN101", "ANN102", "COM812", "D203", "D213", "S603"]
 
 [tool.ruff.lint.per-file-ignores]
 "tests/*" = ["S101"]
+"tests/generate_fixtures/*" = ["T201"]
```

Comment:
I think it's worth catching leftover `print()` calls that were inserted for debugging purposes. We might want to roll this back unless there's something I'm missing.